src/data/multiset/sort.lean

```diff
@@ -50,7 +49,9 @@ list.merge_sort_singleton r a
 
 end sort
 
-instance [has_repr α] : has_repr (multiset α) :=
-⟨λ s, "{" ++ string.intercalate ", " ((s.map repr).sort (≤)) ++ "}"⟩
+meta instance [has_repr α] : has_repr (multiset α) :=
+⟨λ s, "{" ++ string.intercalate ", "
+    (@multiset.sort string (λ s₁ s₂, s₁.to_list ≤ s₂.to_list)
+      undefined undefined undefined undefined (s.map repr) ) ++ "}"⟩
```

Comment:
If we're going to make this meta we may as well not sort it; the sorting is usually useless anyway
```suggestion
⟨λ s, "{" ++ string.intercalate ", " (s.unquot.map repr) ++ "}"⟩
```